static/js/place/place_autocomplete.ts

```diff
@@ -19,6 +19,53 @@ import { getPlaceDcids } from "../utils/place_utils";
 let ac: google.maps.places.Autocomplete;
 let acs: google.maps.places.AutocompleteService;
 
+// Some India cities have limited data, override to show the corresponding district page.
```

Comment:
can we put a todo to move this to flask? (so we do the override in the place2dcid call - to catch all our place searches, and to reduce the size of our js).

Reply:
Moved in this PR